style.css

```diff
@@ -0,0 +1,77 @@
+.toolbar {
+  /* Welcome */
+  height: 28px;
+  left: 12px;
+  top: 10px;
+
+  /* Theme 2 / Subhead 1 */
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+
+  /* identical to box height, or 140% */
+  display: flex;
+  flex-wrap: wrap;
+
+  /* Theme 2 / Neutral 1 */
+  color: #28352f;
+
+  /* Inside auto layout */
+  margin: 0% 0%;
+
+  /* changing list style */
+  list-style: none;
+}
+
+.toolbar .welcome {
+  flex: 1;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+}
+
+.headline p {
+  /* Supporting Text */
+  width: 327px;
+  height: 164px;
+  left: 24px;
+  top: 314px;
+
+  /* Theme 2 / Body Reguler 1 */
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  margin-top: 0%;
+  margin-bottom: 0%;
+
+  /* Neutrals/N800 */
+  color: #172b4d;
+}
+
+.headline span {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+
+  /* Neutrals/N500 */
+  margin-top: 0;
+  color: blue;
+}
+
+.headline img {
+  margin-top: 10px;
+  width: fit-content;
+}
+
+body {
+  background-image: url("/header-shapes.png");
```

Comment:
- [ ] Please remove **"/"** from "/header-shapes.png". If you have the **"/"** on the image path, The image won't show in the live version of your page. You can check [this link](https://www.w3schools.com/html/html_filepaths.asp) to see HTML path. :+1: